duckduckgo/macos-browser
# support blank tab (WIP)

Task/Issue URL: https://app.asana.com/0/1199230911884351/1201228570536147
Tech Design URL: 
CC:

**Description**:

Allows clicking links with target="_blank" to open new tabs for domains that are not the current website.   Especially important for mail clients.

**Steps to test this PR**:
1. Visit https://falkirkrpg.org.uk/noxwall/target-blank.html
1. Click on link - should open blank tab

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
